apps/www/components/Payment/Form/ApplePay.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+
+// Check if the website is running in a secure document.
+function isInSecureDocument(): boolean {
```

Comment:
`isInSecureWindow` or `isInSecureClientWindow` would make it more obvious that this only return true on the client.